src/components/Toggle.js

```diff
@@ -26,6 +26,7 @@ const ToggleContainer = styled.button`
 
 const Toggle = ({ type, toggleTheme }) => (
   <ToggleContainer onClick={toggleTheme}>
+    <div> Toggle Beautiful Theme !</div>
```

Comment:
You can add style in themes to this child, because as I can see, there is black text color.

Reply:
yes we will add styles in themes for actual components. this is for testing the theming works across routes

Reply:
So I cannot accept it. If it will go to master branch, then we shouldn't use `just for testing purpose` concept